packages/evmosjs/src/tests/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:1.20.0-bullseye AS build-env
+
+WORKDIR /go/src/github.com/evmos/
+
+RUN git clone "https://github.com/evmos/evmos.git"
+
+WORKDIR /go/src/github.com/evmos/evmos
+
+RUN apt-get update -y
+
+RUN git checkout main
```

Comment:
did you try it without this line? I think is redundant, 'cause by default should be on `main`
```suggestion
```

Reply:
Also I think would be better to use the latest release for these tests, so we can make sure we didn't break anything on either side (evmos & evmosjs) when we launch a new release

Reply:
I'm setting it to main for now, since we don't have a dedicated `latest` branch. As we create new release branches, we can switch this to test the latest. Until then, main is the best approximation.